src/porepy/models/contact_mechanics_biot_model.py

```diff
@@ -211,8 +211,8 @@ def _set_mechanics_parameters(self) -> None:
 
     def _set_scalar_parameters(self) -> None:
         tensor_scale = self.scalar_scale / self.length_scale**2
-        mass_weight = 1 * self.scalar_scale
         for sd, data in self.mdg.subdomains(return_data=True):
+            mass_weight = self._compressibility(sd) * self.scalar_scale
```

Comment:
Shouldn't this be the biot modulus?

Reply:
I don't think so. This is the factor multiplying $\frac{\partial p}{\partial t}$, or after time discretization, the one multiplying $\left(p^{n} - p^{n-1}\right)$.

This is used in: https://github.com/pmgbergen/porepy/blob/5ba4ce0fa93d33c6cad332663c628db1e249c1a7/src/porepy/models/contact_mechanics_biot_model.py#L845

Reply:
Method was renamed to `_storativity` instead of `_compressibility`.